LoopKit/Persistence/Model.xcdatamodeld/Model.xcdatamodel/contents

```diff
@@ -55,10 +55,10 @@
         <attribute name="createdAt" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
         <attribute name="endDate" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
         <attribute name="hasLoopKitOrigin" attributeType="Boolean" usesScalarValueType="YES" syncable="YES"/>
+        <attribute name="programmedTempBasalRate" optional="YES" attributeType="Double" usesScalarValueType="NO" elementID="scheduledTempBasalRate" syncable="YES"/>
```

Comment:
So you aren't creating a new model version to allow Core Data to do an automatic migration?

Reply:
Or was the `scheduledTempBasalRate` field never released?

Reply:
Core data will perform the migration by looking at the data model hash; it doesn't use the version identifier. The `scheduledTempBasalRate` is used as the identifier for renaming that field to `programmedTempBasalRate`